plugins/outputs/iotdb/README.md

```diff
@@ -0,0 +1,121 @@
+the cloud. Due to its light-weight architecture, high performance and rich
+feature set together with its deep integration with Apache Hadoop, Spark and
+Flink, Apache IoTDB can meet the requirements of massive data storage,
+high-speed data ingestion and complex data analysis in the IoT industrial
+fields.
+
+For more details consult the [Apache IoTDB website](https://iotdb.apache.org)
+or the [Apache IoTDB GitHub page](https://github.com/apache/iotdb).
+
+## Getting started
+
+Before using this plugin, please configure the IP address, port number,
+user name, password and other information of the database server,
+as well as some data type conversion, time unit and other configurations.
+
+Please see the [configuration section](#Configuration) for an example
+configuration.
+
+## Please pay attention to these points
+
+1. IoTDB (version 0.13.x or older) **DO NOT support unsigned integer**.
+There are three available options of converting uint64, which are specified by
+parameter `uint64_conversion`.
+
+   - `int64_clip`, default option. If an unsigned integer is greater than
+   `math.MaxInt64`, save it as `int64`; else save `math.MaxInt64`
+   (9223372036854775807).
+   - `int64`, force converting an unsigned integer to a`int64`,no mater
+   what the value it is. This option may lead to exception if the value is
+   greater than `int64`.
+   - `text`force converting an unsigned integer to a string, no mater what the
+   value it is.
+
+2. IoTDB supports a variety of time precision. You can specify which precision
+you want using the `timestamp_precision` option. Default is `nanosecond`.
+Other options are "second", "millisecond", "microsecond".
+
+3. Till now, IoTDB can not support Tag indexing well. To see current process
+   method, please refer to [InfluxDB-Protocol Adapter](
+   https://iotdb.apache.org/UserGuide/Master/API/InfluxDB-Protocol.html).
+   There are two available options of converting tags, which are specified by
+   parameter `convert_tags_to`:
+
+   - `fields`. Treat Tags as measurements. For each Key:Value in Tag,
+   convert them into Measurement, Value, DataType, which are supported in IoTDB.
+   - `device_id`, default option. Treat Tags as part of device id. Tags
+   is subtree of 'Name'.
+
+   For example, there is a metric:
+
+   `Name="root.sg.device", Tags={tag1="private", tag2="working"}, Fields={s1=100, s2="hello"}`
+
+   - `fields`, result: `root.sg.device, s1=100, s2="hello", tag1="private", tag2="working"`
+   - `device_id`, result: `root.sg.device.private.working, s1=100, s2="hello"`
+
+## Testing
+
+Testing contains: network connection, error correction, datatype conversion,
+data writing.
+
+Use `-short` option to make testing shorter. This mode skips integration tests.
+
+## Configuration
+
+```toml @sample.conf
+# Save metrics to an IoTDB Database
+[[outputs.iotdb]]
+  ## Configuration of IoTDB server connection
+  host = "127.0.0.1"
+  # port = "6667"
```

Comment:
I' sorry. It's my bad that I did not run `make docs`. I will run it after next commit.

I guess it will automatically correct my `readme` and attach `sample.conf`, right? However, it did not worked, perhaps because of the local settings of windows. I need to manually modify the `makefile` to make it run, but after running the command, nothing really happened. 

But I have made sure that the contents in `readmd` are same with  `sample.conf`.

Thanks

Reply:
It worked, the REAME is adapted... Yes, the `@sample.conf` statement in the toml section will indicate that you want to embed that file at this location to the tool I wrote (`tools/readme_config_includer`). Let me know (i.e. create an issue) if you have trouble on Windows!